app/jsx/components/JumpComp.jsx

```diff
@@ -12,7 +12,7 @@ class JumpComp extends React.Component {
   render() {
     return (
       <div className="c-jump">
-        <a id="c-jump__label" href="">Article</a>
+        <Link id="c-jump__label" to="#" onClick={(e)=>this.handleClick(e, 1)}>Article</Link>
```

Comment:
I realize this is just temporary until we talk through the permutations of URL hash, back button, and tabs on the item page.